### PR TITLE
Feat/ show inactive shifts in barista dashboard and modify delete shift logic

### PR DESCRIPTION
**Pull request by Neuroleptique**
Implemented 2 related features: 
  1. Show past shifts on barista dashboard, if they put themself available for it
  2. Auto delete shift if both barista and cafe remove it from their dashboard

Question about this implementation:
- Should the modification be divided into 2 separate PRs?
- ~~Is keeping the old deleteShift function commented out necessary?~~ (deleted after reviewing PR review comments for another PR)

### CODE REVIEW

File: controllers/accounts.js [hunk 1]
```diff
@@ -20,18 +20,33 @@ module.exports = {
...
+              { date: { $lt: today }},
+              { activeStatus: false }
+            ]}
+          ]
+          
+        })
+
+        const activeShiftPoster = activeShiftData.map( s => s.cafeUserName )
+        const pastShiftPoster = pastShiftData.map( s => s.cafeUserName)
+        const shiftPoster = activeShiftPoster.concat(pastShiftPoster)
```
Comment (WillowP) [inline controllers/accounts.js:41]: I'm a little confused by these variable names. They are named as if they are single values or objects, but I think they are arrays. If so, I think just making the var names plural would help. Also you might consider renaming `shiftPoster` to `allShiftPosters` to be more expressive.

**Comment by WillowP** [comment]
I do think you could have separated this into 2 PRs, since the 2 features are not really related in any way that I can see.

**Review by WillowP — CHANGES_REQUESTED** [review]
Added one minor code style comment. I think splitting this into 2 PRs is a good idea though :)

File: controllers/accounts.js (hunk 1 above)
Comment (Neuroleptique) [reply controllers/accounts.js:41]: This change will be applied in Feat/baristaShowPastShift/new as I divide this PR into 2 separated PR.

**Pull request closed without merging**